### ISSUE TITLE
Version of Dynamo needed to build this

### ISSUE THREAD

**Issue by AlexFielder** [frankfralick/DynamoInventor#3]
I am having trouble getting this to build- my guess is because I have the latest version of Dynamo installed? (which at the time of writing is 0.9.0)

**Comment by jluebker** [comment]
you are getting a bunch of errors in the build as well? .. 516 errors and 4 warnings? did you see the comments in the DynamoInventor.cs?

```
  //Assembly.LoadFrom(@"C:\Projects\Dynamo\Dynamo\bin\AnyCPU\Debug\DynamoCore.dll");
        // Even though this method has no dependencies on DynamoCore, resolution of DynamoCore is 
        // failing prior to the handler to AssemblyResolve event getting registered.  
```

**Comment by jluebker** [comment]
from this post: https://github.com/DynamoDS/Dynamo/issues/3387

it seems like we might need the dynamo src as well...

**Comment by jluebker** [comment]
![image](https://cloud.githubusercontent.com/assets/4041748/11322098/c810741a-908f-11e5-96a6-f35e165f5afb.png)

look at this..

**Comment by AlexFielder** [comment]
The problem is that I have the latest source and it mostly builds but I think there is some part that's no longer in the latest source.

**Comment by jluebker** [comment]
Possible solutions: point it to the DynamoCore.dll in an actual build... if 9 is not working maybe it would work to point it to an older version like 8 or 7. I'm fumbling my way through this code for the first time...but it sounds like the first step would be to break the dependency on the default output DynamoCore.dll where ever it lands when you compile the Dynamo dll and point it to where it should be in an actual installed build.... maybe

**Comment by jluebker** [comment]
http://dynamobim.com/forums/topic/debugging-with-visual-studio/ take a look at this similar problem other are having with forks from the Dynamo build

**Comment by AlexFielder** [comment]
I've just downloaded the 0.9.0 version and I also have the "DynamoPathManager" class missing as per your image above - any ideas what is missing?

**Comment by jluebker** [comment]
Yeah I think it is referencing the dynamo 7 folder. I’d have to go back and look at my string of emails with Frank.

Jeremy Luebker
Arch II
t 415.546.2957
c 510.881.3682
Perkins+Will

**Comment by AlexFielder** [comment]
If you could look into it that would be great, thanks!

**Comment by jluebker** [comment]
Honestly, I don’t know visual studio that well, but I know enough code to read and understand (I’m more of a python guy in rhino, when it comes to coding). But from what I can tell  in the DynamoInventor.cs file, it is looking for the DynamoCore.dll in C:\Projects\Dynamo\Dynamo\bin\AnyCPU\Debug. You can find the DynamoCore.dll in C:\Program Files\Dynamo 0.9. I tried simply changing this link but got some additional errors preventing me from compiling the dynamoinventor.dll. I haven’t had a chance to stumble through that.

The line of code you want to look at is all the way at the bottom of the DynamoInventor.cs, try changing the association of the dynamocore.dll. if it works for you, maybe you can tell me what I’m missing….

Good luck, let me know how it goes…

Jeremy Luebker
Arch II
t 415.546.2957
c 510.881.3682
Perkins+Will
